conversion/grad_avg.py

```diff
@@ -25,7 +25,7 @@ def grad_avg(imgFile, outFile, bvalFile= None, bvecFile= None):
         bvecs= read_bvecs(bvecFile)
 
         img= nib.load(imgFile)
-        data = img.get_data()
+        data = img.get_fdata()
```

Comment:
I have moved to `get_fdata()` in my other projects, did not realize it was still the old one here.

Reply:
Just trying to use your good work to save me some time.   :)

I'd love to run the code beautifier 'Black' on the code to make it consistent internally, and consistent with many other medical imaging packages.  Are you supportive of that?

Reply:
Code beautifier shouldn't change my algorithm. So feel free to run and propose changes.